docs/views.rst

```diff
@@ -119,7 +119,27 @@ You can then resume the passage of time with the
     >>> wwt.play_time()
 
 You can also change the rate of passage of time by passing a ``rate`` argument
-to :meth:`~pywwt.BaseWWTWidget.play_time` - for example to speed things up by
+to :meth:`~pywwt.BaseWWTWidget.play_time` - for example, to speed things up by
 a factor of 10000, use::
 
     >>> wwt.play_time(rate=10000)
+    
+It is also possible to specify a time in the viewer yourself through the 
+:meth:`~pywwt.BaseWWTWidget.set_current_time` method as long as the time
+provided is a :class:`~datetime.datetime` or an
+`astropy <http://docs.astropy.org/en/stable/time/index.html>`_
+:class:`~astropy.time.Time` object::
+
+    >>> from astropy.time import Time
+    >>> my_time = Time('2017-03-09')
+    >>> wwt.set_current_time(my_time)
+    
+(To bring the viewer back to real time, call 
+:meth:`~pywwt.BaseWWTWidget.set_current_time` without an argument.)
+
+If you're unsure of where in time you've landed, the
+:meth:`~pywwt.BaseWWTWidget.get_current_time` method provides a printout of the
+viewer's current time in UTC::
```

Comment:
The printout only happens in an interactive environment where you don't assign the value to a variable. This should read "the get_current_time method returns the viewer's current time as an :class:`astropy.time.Time` object in the UTC timezone.

Reply:
Mostly adopted, but I thought that "in (the) UTC timezone" was redundant (e.g. "EDT time") and just saying "in UTC" seems to be common. If you feel strongly about it I can push another commit.